src/cdk/observers/private/shared-resize-observer.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {inject, Injectable, NgZone, OnDestroy} from '@angular/core';
+import {Observable, Subject} from 'rxjs';
+import {filter, shareReplay, takeUntil} from 'rxjs/operators';
+import {Platform} from '@angular/cdk/platform';
+
+/**
+ * Handler that logs "ResizeObserver loop limit exceeded" errors.
+ * These errors are not shown in the Chrome console, so we log them to ensure developers are aware.
```

Comment:
This seems counter-intuitive. Maybe Angular's `ErrorHandler` or Zone.js are being weird?

Reply:
I think they did it this way because the error can safely be ignored without breaking your app. Not sure why they didn't just make it a warning though. It's not because of Angular, here's a plain JS example: https://jsfiddle.net/1fz3avsj/ (open the real console, not the emulated console).